dask/dataframe/partitionquantiles.py

```diff
@@ -425,12 +423,22 @@ def percentiles_summary(df, num_old, num_new, upsample, state):
     elif is_datetime64_dtype(data.dtype) or is_integer_dtype(data.dtype):
         interpolation = "nearest"
 
-    # FIXME: pandas quantile doesn't work with some data types (e.g. strings).
-    # We fall back to an ndarray as a workaround.
+    # FIXME: Series.quantile doesn't work with some data types (e.g. strings).
+    # We try using DataFrame.quantile(..., method="table") as a workaround.
     try:
         vals = data.quantile(q=qs / 100, interpolation=interpolation).values
     except (TypeError, NotImplementedError):
-        vals, _ = _percentile(array_safe(data, like=data.values), qs, interpolation)
```

Comment:
**TODO**: There *may* still be cases where we **need** `_percentile`. Need to check this.